libraries/chain/db_maint.cpp

```diff
@@ -393,10 +393,12 @@ void database::update_active_sons()
       }
    }
 
-   const chain_property_object& cpo = get_chain_properties();
-   auto sons = sort_votable_objects<son_index>(std::max(son_count*2+1, (size_t)cpo.immutable_parameters.min_son_count));
+   //const chain_property_object& cpo = get_chain_properties();
```

Comment:
Why is these two lines commented out ? If its anticipating changes in the future, please add the reasoning as a comment above.

Reply:
Removed